src/pls/fs/git.py

```diff
@@ -90,10 +90,18 @@ def get_git_statuses(git_root: Path) -> dict[Path, str]:
         status = line[0:2]
 
         components: list[str] = shlex.split(line[3:])
+        if len(components) == 0:  # It might be windows
+            win_components: list[str] = shlex.split(line[3:], posix=False)
+            if len(win_components) != 0:  # If different result use those
+                components = win_components
```

Comment:
I'm unfamiliar with `shlex` as much but wouldn't this be simpler?
```suggestion
        components: list[str] = shlex.split(line[3:], posix=(os.platform!="win32"))
```

Reply:
Yep, I just changed it.
I did it that way for less imports, but it's probably faster using that.

Reply:
Does this change fix all the errors? The main reason for using `shlex.split` instead of regular `.split()` was to prevent the code from breaking when there's a space in the file name and Git wraps it in quotes.

Reply:
Although this might fix the error, I suppose the presence of quotes around the string when `posix=False` will lead to bugs.

<img width="361" alt="Screenshot 2022-02-14 at 4 04 11 PM" src="https://user-images.githubusercontent.com/16580576/153861084-17eea468-ffdc-49cf-b642-719fb9f1ae3e.png">

Reply:
![image](https://user-images.githubusercontent.com/37764578/153929578-898160ef-63e3-497a-b366-c321da005be2.png)
It seems fine with spaces

Reply:
![image](https://user-images.githubusercontent.com/37764578/153930431-fb836ecc-9550-4efa-bcb8-8a5c2ab52bf4.png)
But it doesn't seem to get the status correctly

Reply:
![image](https://user-images.githubusercontent.com/37764578/153949797-efd34083-07d3-4982-8441-e133c7380900.png)
Seems to be fine now.